src/main/scala/com/oysterbooks/scavro/plugin/AvroCodegenPlugin.scala

```diff
@@ -10,10 +10,15 @@ object AvroCodegenPlugin extends AutoPlugin {
     val avroSchemaFiles = settingKey[Seq[File]]("Avro schema files to compile")
     val avroProtocolFiles = settingKey[Seq[File]]("Avro Protocol files to compile")
     val avroIDLFiles = settingKey[Seq[File]]("Avro IDL files to compile")
+    val avroDataFiles = settingKey[Seq[File]]("Avro datafiles to compile")
 
     val showAvroCompilerOutput = settingKey[Boolean]("Hides messages from avro compiler")
 
     val avroCodeOutputDirectory = settingKey[File]("Location where generated Java code will be placed")
+    val scalaCodeOutputDirectory = settingKey[File]("Location where generated Scala wrapper code will be placed")
+
+    val scalaCustomTypes = settingKey[Map[String, String]]("Customize Avro to Scala type map by type")
+    val scalaCustomNamespace = settingKey[Map[String, String]]("Custom namespace of generated Scala wrapper code")
```

Comment:
Is it possible to infer this, or at least provide a reasonable default value?

Also both of these parameters need names that make it more clear what they do.  In a large project `scalaCustomTypes` doesn't give any indication that it's related to the scavro plugin or involves avro serialization.